test/constraint/spec.test.ts

```diff
@@ -858,6 +858,26 @@ describe('constraints/spec', () => {
       assert.isFalse(SPEC_CONSTRAINT_INDEX['omitRaw'].satisfy(specM, schema, DEFAULT_QUERY_CONFIG));
     });
 
+    it('should not exclude aggregate if it is still an enum spec', () => {
```

Comment:
What does "exclude aggregate" mean?  It reads very confusing.  

The method returns true or false.  Why don't you use consistent message as others?
e.g., `'It should return true if there is an enum spec aggregate'`.
 
(As mentioned this morning, you definitely need to spend more time with test description as I have been correcting them for you for almost every PRs.)